source/clear-linux/reference/collaboration/community/conduct.rst

```diff
@@ -0,0 +1,98 @@
+welcomes and supports people of all backgrounds and identities. This
+includes, but is not limited to, members of any race, ethnicity, culture,
+national origin, color, immigration status, social and economic class,
+educational level, sex, sexual orientation, gender identity and expression,
+age, size, family status, political belief, religion, and mental and physical
+ability.
+
+**Be considerate.** Our work will be used by other people and we, in turn,
+will depend on the work of others. Any decision we take will affect users and
+colleagues. We should take those consequences into account when making
+decisions. Remember, we are a world-wide community and we have a global base
+of users and contributors. Even if it is not obvious at the time, our
+contributions to the |CL| project will impact the work of others.
+
+**Be respectful.** Not all of us will agree all the time but disagreement is
+no excuse for poor behavior and poor manners. We might all experience some
+frustration now and then but we cannot allow frustration to turn into a
+personal attack. Remember, a community where people feel uncomfortable or
+threatened is not productive. Members of the |CL| community should be
+respectful when dealing with other contributors, people outside of the
+project, and users.
+
+**Collaborate openly.** Collaboration is central to the |CL| project and to
+the larger free software community. This collaboration involves individuals
+working within teams and working with other projects outside of the |CL|
+community. This collaboration reduces redundancy and improves the quality of
+our work. Internally and externally, we should always be open to
+collaboration. Wherever possible, we should work closely with upstream and
+downstream projects and others in the free software community to coordinate
+our technical, advocacy, documentation, and other work. Our work should be
+done transparently and we should involve as many interested parties as early
+as possible. If we decide to take a different approach, we will let them know
+early, document our work, and inform others regularly of our progress. Do not
+create private forms of communication that take away transparency or exclude
+other contributors and collaborators.
+
+**When we disagree, try to understand why.** Disagreements, both social and
+technical, happen all the time and the |CL| community is no exception.
+Resolve disagreements and differing views constructively. Remember, we are
+different. The strength of the community comes from its diversity: people
+from a wide range of backgrounds. Different people have different
+perspectives on issues. Being unable to understand why someone holds a
+viewpoint does not mean they are wrong. Focus on helping resolve issues and
+learning from mistakes. Resolve disagreements and differing views
+constructively and with the help of the community and community processes.
+
+**When we are unsure, we ask for help.** Nobody knows everything and nobody
```

Comment:
This title and the one previous is inconsistent with others by using "we":
When unsure, ask for help

Reply:
done